perfkitbenchmarker/aws/aws_virtual_machine.py

```diff
@@ -278,19 +285,24 @@ def CreateScratchDisk(self, disk_spec):
       disk_spec: virtual_machine.BaseDiskSpec object of the disk.
     """
     # Instantiate the disk(s) that we want to create.
-    if disk_spec.disk_type == disk.LOCAL:
-      disks = []
-      for _ in range(disk_spec.num_striped_disks):
-        local_disk = aws_disk.AwsDisk(disk_spec, self.zone)
-        local_disk.device_letter = chr(ord(DRIVE_START_LETTER) +
-                                       self.local_disk_counter)
+    disks = []
+    for _ in range(disk_spec.num_striped_disks):
+      data_disk = aws_disk.AwsDisk(disk_spec, self.zone)
+      if disk_spec.disk_type == disk.LOCAL:
+        data_disk.device_letter = chr(ord(DRIVE_START_LETTER) +
+                                      self.local_disk_counter)
+        # Local disk numbers start at 1 (0 is the system disk).
+        data_disk.disk_number = self.local_disk_counter + 1
         self.local_disk_counter += 1
-        disks.append(local_disk)
-      if self.local_disk_counter > self.max_local_disks:
-        raise errors.Error('Not enough local disks.')
-    else:
-      disks = [aws_disk.AwsDisk(disk_spec, self.zone)
-               for _ in range(disk_spec.num_striped_disks)]
+        if self.local_disk_counter > self.max_local_disks:
+          raise errors.Error('Not enough local disks.')
+      else:
+        # Remote disk numbers start at 1 + max_local disks (0 is the system disk
+        # and local disks occupy 1-max_local_disks).
```

Comment:
"1-max_local_disks" confused me (looked like subtraction). How about something like "[1, max_local_disks)" or expanding it out as a sentence?